apps/stripe/src/modules/app-result/types.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  AuthorizationActionRequiredResult,
+  ChargeActionRequiredResult,
+} from "./action-required-result";
+import { AuthorizationFailureResult, ChargeFailureResult } from "./failure-result";
+import { AuthorizationRequestResult, ChargeRequestResult } from "./request-result";
+import { AuthorizationSuccessResult, ChargeSuccessResult } from "./success-result";
+
+export type AppResult =
```

Comment:
Im not sure if AppResult is a good name? App is quite abstract.

The common thing is Transaction right? So why not TransactionResult?

Reply:
Done in https://github.com/saleor/apps/pull/1885/commits/dbc20052bba0de0331479f43bd84ebd26f476270